metpetdb_api/api/samples/v1/renderers.py

```diff
@@ -0,0 +1,139 @@
+from rest_framework_csv import renderers as r
+
+
+class SampleCSVRenderer (r.CSVRenderer):
+    header = ['number',
```

Comment:
Following Python conventions, these member variables should be initialized as part of a constructor. i.e 
```
def __init__(self):
  self.header = [....]
  self.labels = {..}
  self.num_regions = 0
   .
   .
   .
     
```